### ISSUE TITLE
Form is missing labels

### ISSUE THREAD

**Issue by matthall8** [FACN4/BSN_week1-group-project#21]
The website fails the accessibility test on the grounds of missing labels for the input fields on the contact form.

![image](https://user-images.githubusercontent.com/9437789/42310059-1a6605cc-8043-11e8-8137-a3203f01af82.png)

**Comment by BenNewman90** [comment]
Thanks. Resolved.
